easybuild/easyconfigs/g/gpustat/gpustat-0.5.0-fosscuda-2018b-Python-2.7.15.eb

```diff
@@ -0,0 +1,44 @@
+easyblock = 'PythonBundle'
+
+name = 'gpustat'
+version = '0.5.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/wookayin/gpustat'
+description = 'dstat-like utilization monitor for NVIDIA GPUs'
+
+toolchain = {'name': 'fosscuda', 'version': '2018b'}
+
+dependencies = [
+    ('Python', '2.7.15'),
+    ('CUDA', '9.2.88'),
```

Comment:
this is not necessary, it's already included in the `fosscuda` toolchain

Reply:
Fixed